### DISCUSSION TITLE
Multiple hosts on different ports

### DISCUSSION THREAD

**Discussion by gusarov** [nginx-proxy/nginx-proxy#2295] [Q&A] [answered]
Hello, I'd like to host example1.com that is exposed as port 1024 and example2.com that is exposed as port 1025 on same container. Based on documentation I can set multiple hosts to a VIRTUAL_HOST=example1.com,example2.com but how I can instruct what host connects to what port? Is it not possible to do with nginx-proxy?

**Comment by frnzhop** [comment]
+1

**Comment by buchdag** [comment, marked as answer]
https://github.com/nginx-proxy/nginx-proxy/tree/main/docs#multiple-ports
